.github/workflows/lib-codeql.yaml

```diff
@@ -25,11 +25,11 @@ jobs:
         check-latest: true
 
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@2c779ab0d087cd7fe7b826087247c2c81f27bfa6 # v3
+      uses: github/codeql-action/init@294a9d92911152fe08befb9ec03e240add280cb3 # v3
```

Comment:
The `codeql-action` is updated roughly every 2 weeks and it's massively used. I think it's not in our resources to keep monitoring the changes so I'd simply consume `v3` and only review changes when `v4` is introduced.

Reply:
(and yes, I understand the risk, I just don't think the benefit is worth the extra effort)

Reply:
I'm fine making dependabot updates for actions monthly now that all actions are updated